server/internal/usecase/interfaces/integration.go

```diff
@@ -47,6 +47,8 @@ type Integration interface {
 	Create(context.Context, CreateIntegrationParam, *usecase.Operator) (*integration.Integration, error)
 	Update(context.Context, id.IntegrationID, UpdateIntegrationParam, *usecase.Operator) (*integration.Integration, error)
 	Delete(context.Context, id.IntegrationID, *usecase.Operator) error
+	// DeleteMany deletes integrations in batch
+	DeleteMany(context.Context, id.IntegrationIDList, *usecase.Operator) error
```

Comment:
```suggestion
	DeleteMany(context.Context, id.IntegrationIDList, *usecase.Operator) error
```
no need for comment when its obvious